src/common/utils.cpp

```diff
@@ -1,6 +1,8 @@
 #include "duckdb.hpp"
 #include "iceberg_utils.hpp"
 #include "fstream"
+#include "../include/storage/irc_table_entry.hpp"
```

Comment:
No relative includes please

Reply:
Still call it out though, otherwise they will accidentally end up in there